### DISCUSSION TITLE
Authorization Key in HTTP-Header

### DISCUSSION THREAD

**Discussion by werkhausen** [QuickSander/homebridge-http-rgb-push#7] [General]
Hello everyone,

it's possible to set / edit the Header of the request to add a Authorization-Key?

Best Regards

Malte

**Comment by QuickSander** [comment]
Hi,

The original where I forked from contains user name and password settings for http basic authentication requests. 

I never tried it myself, but the code is still there. See also the README and the overview table with all options.
